pkg/client/client.go

```diff
@@ -70,15 +70,24 @@ func (cl *ETHClient) GetTransactionReceipt(ctx context.Context, txHash common.Ha
 		reason, err := r.GetRevertReason()
 		return &r.Receipt, false, fmt.Errorf("revert: %v(parse-err=%v)", reason, err)
 	} else {
-		return &r.Receipt, false, fmt.Errorf("failed to execute a transaction: %v", r)
+		errPrefix := "failed to execute a transaction"
+		if enableDebugTrace {
+			to, revertReason, err := cl.DebugTraceTransaction(ctx, txHash)
+			if err != nil {
+				return &r.Receipt, false, fmt.Errorf("%s: %v, debug_transaction error: %v", errPrefix, r, err)
+			}
+			return &r.Receipt, false, fmt.Errorf("%s: %v, contract: %s, revert reason: %v", errPrefix, r, to, revertReason)
```

Comment:
could you add TxHash to the log

Reply:
<img width="1439" alt="Screenshot 2023-08-01 at 13 23 43" src="https://github.com/datachainlab/ethereum-ibc-relay-chain/assets/126585618/c47f5ca4-b4a9-4220-9c9e-a16f275284dc">
Add txHash to the log
https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/10/commits/53a99c31942d630704e8abb75c5d2899fe179aec